packages/pieces/src/lib/apps/drip/actions/apply-tag-to-subscriber.action.ts

```diff
@@ -0,0 +1,41 @@
+import { httpClient } from "../../../common/http/core/http-client";
+import { HttpMethod } from "../../../common/http/core/http-method";
+import { HttpRequest } from "../../../common/http/core/http-request";
+import { createAction } from "../../../framework/action/action";
+import { Property } from "../../../framework/property";
+import { dripCommon } from "../common";
+
+export const dripApplyTagToSubscriber = createAction({
+    name: 'apply_tag_to_subscriber',
+    description: 'Apply a tag to a subscriber',
+    displayName: 'Apply a tag to subscriber',
+    props: {
+        authentication: dripCommon.authentication,
+        account_id: dripCommon.account_id,
+        subscriber: dripCommon.subscriber,
+        tag: Property.ShortText({
+            displayName: "Tag",
+            required: true,
+            description: "Tag to apply"
+        })
+    },
+    sampleData: {},
```

Comment:
Provide sample data

Reply:
This is the response an empty object. https://developer.drip.com/#apply-a-tag-to-a-subscriber